example/index.html

```diff
@@ -22,12 +22,13 @@
         <img id="floorimg" src="assets/textures/grasslight-big.jpg">
         <img id="floorimg2" src="assets/textures/grasslight-big.jpg">
       </a-assets>
-<!--      <a-entity id="soundcube" position="0 0 -10" rotation="45 30 0"
-          sound__1="autoplay: true; src: assets/sounds/321103__nsstudios__blip1.wav;"
-          sound__2="autoplay: true; src: assets/sounds/Laser-SoundBible.com-602495617.com-602495617.ogg;"
+      <a-entity id="soundcube" position="0 0 -10" rotation="45 30 0"
```

Comment:
What you mean, that it won't play it automatically? Right now it doesn't have `autoplay: true`  or any event assigned so it won't play at all. Still I'll remove it from the example, I forgot to delete it before pushing